Ginger/GingerCore/Platforms/TargetApplication.cs

```diff
@@ -63,5 +63,11 @@ public override string ItemName
                 return;
             }
         }
+
+        [IsSerializedForLocalRepository]
+        public Agent.eAgentType TargetAgentType { get; set; }
+
```

Comment:
how TargetAgentType is related to TargetApplication? we said we will remove the dependncy on the Target Application and the plugins dependency will be added to the Runner based on added Actions no?